vagrant/lib/forklift/box_factory.rb

```diff
@@ -37,6 +38,10 @@ def load_box_file(file)
 
     def process_boxes!(boxes)
       boxes.each do |name, box|
+        box_config = Settings.new.settings['boxes']
+
+        next if box_config.key?('exclude') && box_config['exclude'].any? { |exclude| name.include?(exclude) }
```

Comment:
@ehelms how do you feel about allowing regex with `name.match(/#{exclude}/)`? That would allow for some more fine tuning over what is allowed, but still work for a simple include.

Reply:
Done.